src/Widgets/Tabs/Info.hs

```diff
@@ -174,8 +175,16 @@ renderPropVal val = mdo
   loseFocusE <- getTxt (\t -> () <$ ffilter not (updated $ t^.textInput_hasFocus))
   let saveE = leftmost [enterE, loseFocusE]
   (editBtn, _) <- elClass' "span" "icon" $ text "edit"
+  let editBtnClickE = domEvent Click editBtn
+
+  -- focus textInput after it's created
+  let doFocus (Just t) = liftIO $ focus $ _textInput_element t
+      doFocus Nothing  = return ()
+  editBtnClickDelayedE <- delay 0.1 editBtnClickE
+  performEvent_ $ doFocus <$> (current mTxtInputD <@ editBtnClickDelayedE)
```

Comment:
So, just using `doFocus <$> updated (fmapMaybe id mTxtInputD) ` does not work? Do you mean `delay` does not work in this case?